src/Resources/model/Machine.py

```diff
@@ -58,6 +59,12 @@ def add_meta(self, name, value):
             self.bridge = self.lab.get_or_new_link(BRIDGE_LINK_NAME)
             return
 
+        if name == "sysctl":
+            # Assume format [str]=[int]
+            parts = value.split('=')
+            self.sysctls[parts[0].strip()] = int(parts[1].strip())
```

Comment:
A check on the prefix of the sysctl would be nice.
In #82, we discussed the possibility to enable only the `net.` namespace instead of everything.
So, something like:
```python
if parts[0].strip().startswith('net.')
```

Also, it would be great to check if `=` is in the string before splitting, if not raise an exception such as `ValueError` with a brief explanation.